analyses/cellranger-analysis/j1.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+########################################################################
+# Load modules
+module load python/3.9.9
+module load cellranger/8.0.1
+########################################################################
+
+# Set up running directory
+cd "$(dirname "${BASH_SOURCE[0]}")" 
+
+# If your `project_metadata` is not in `*.txt` file format
+# use the following code line to convert it
+cat /research/dept/dnb/core_operations/Bioinformatics/zakhagrp/snRNASeq_of_DS_in_Df1_mice/project_metadata.tsv | sed 's/,/\t/g' > ./input/project_metadata.txt
```

Comment:
Maybe for extra clarity, you could add a comment here specify that if the project_metadata file doesn't need to be converted, comment out this command